frontend/src/routes/Upload.tsx

```diff
@@ -784,17 +798,21 @@ const MetaDataEdit: React.FC<MetaDataEditProps> = ({ onSave, disabled, knownRole
                     marginBottom: 12,
                     fontSize: 22,
                 }}>{t("manage.my-videos.acl.title")}</h2>
-                <Controller
-                    name="acl"
-                    control={control}
-                    render={({ field }) => <AclSelector
-                        userIsRequired
-                        onChange={field.onChange}
-                        acl={field.value}
-                        knownRoles={knownRoles}
-                        permissionLevels={READ_WRITE_ACTIONS}
-                    />}
-                />
+                <div {...lockedAcl.size > 0 && { inert: "true" }} css={{
+                    ...lockedAcl.size > 0 && { opacity: .7 },
```

Comment:
I think it would be nicer if we wouldn't use an empty map as sentinel value, but rather make the `lockedAcl` a `Map | null` or sth like that. Of course, with one of my prior comments about loading the acl of the series only when it is selected, this might change even more (maybe `Map | "loading" | ...` or sth like that)